lib/util/docker/index.ts

```diff
@@ -7,37 +9,55 @@ export interface DockerOptions {
   envVars?: Opt<Opt<string>[]>;
   cwd?: Opt<string>;
   tag?: Opt<string>;
-  cmdWrap?: Opt<string>;
+  preCommands?: Opt<string[]>;
```

Comment:
Isn't this a separate change? Can we make it into a different PR to keep each PR as minimal as possible?